targets/TARGET_Realtek/TARGET_AMEBA/TARGET_MCU_RTL8710A/TARGET_RTL8710A/PeripheralNames.h

```diff
@@ -0,0 +1,42 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
+ *
```

Comment:
new files should contain SPDX identifier (look at platform files for instance how to)

Reply:
Ok, I understand. I'm going to update.